README.md

```diff
@@ -389,7 +389,8 @@ EditorConfig
 | hitTestSize            | hit test region of corner and line                                 | 20.0                                                         |
 | animationDuration      | auto center animation duration                                     | Duration(milliseconds: 200)                                  |
 | tickerDuration         | duration to begin auto center animation after crop rect is changed | Duration(milliseconds: 400)                                  |
-| cropAspectRatio        | aspect ratio of crop rect                                          | null(custom)                                                 |
+| cropAspectRatio        | aspect ratio of crop rect                                          | null(custom)         
+| initialCropAspectRatio | initial aspect ratio of crop rect                                          | null(custom: initial crop rect will fill the entire image)                                                 |                                        |
```

Comment:
Not well-aligned

Reply:
I don't see the problem can you suggest a code for this please?

Reply:
It's hard to suggest from gthe GitHub website. Maybe you can copy-paste the above line and align them manually?